### DISCUSSION TITLE
about automatically upload  gltf/glb in google model viewer editor

### DISCUSSION THREAD

**Discussion by ghost** [google/model-viewer#3117] [Q&A]
i am junior programmer

Is it possible to automatically upload a gltf/glb file to Google Model Editor after creating a gltf/glb file on my web page?

i make to create glb file in threejs.

after i want to automatically upload google model viewer editor on my web page.

help me please.

this is my site https://okyoungbin.github.io/Threejs_modelViewer/

**Comment by elalish** [comment]
Probably? I suppose you could start by embedding our editor in an iframe. You may  want to hunt down a senior developer to help out.
